packages/ember-cli-eyeglass/README.md

```diff
@@ -31,12 +32,82 @@ const app = new EmberApp(defaults, {
 });
 ```
 
+Given the following layout:
+
+```sh
+app/styles/app.scss
+           other.scss
+           _shared.scss
+```
+
+
+Will result in:
+
+`assets/<app-name>.css` containing the compiled output of `app/styles/app.scss`
+`assets/foo.css` containing the compiled output of `app/styles/foo.scss`
+
+The contents of `app/styles/_shared.scss` will be include only if required, and will appear in the file which imported them.
```

Comment:
You should reference that the underscore indicates that this is a partial, and link to this https://sass-lang.com/guide#topic-4

Reply:
chris added a note to cover this